WhatBackend.EmailRenderService/WhatBackend.EmailRenderService/WhatBackend.EmailRenderService.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Please move sln into WhatBackend\WhatBackend.EmailRenderService.sln. it helps to understand which solutions do we have, when we're in the root folder of our projects